scriptmodules/supplementary/skyscraper.sh

```diff
@@ -44,14 +45,19 @@ function install_skyscraper() {
         'artwork.xml.example2'
         'artwork.xml.example3'
         'artwork.xml.example4'
+        'platforms.json'
+        'screenscraper.json'
+        'mobygames.json'
         'tgdb_developers.json'
         'tgdb_publishers.json'
         'mameMap.csv'
         'aliasMap.csv'
-        'hints.txt'
+        'hints.xml'
+        'docs'
         'import'
         'resources'
         'cache/priorities.xml.example'
+        'VERSION'
```

Comment:
is the `VERSION` really needed ?

Reply:
Not really. I guess that's pretty well covered by `retropie.pkg` innit.